### ISSUE TITLE
Users can write custom notes for files and folders

### ISSUE THREAD

**Issue by Richargh** [MaibornWolff/codecharta#611]
# Feature request

While doing an audit our users like to write down things that they notice for certain buildings or even packages. Currently they do this in a separate editor like this: *"file <filepath> is very big, has a lot of complexity and is changed very often".* Afterwards they then check all their notes and analyze if the same file has different notes from different views. 

What if the user can add custom notes for files and folders in CC? 

## Description

**When** the user clicks on a building or package the metric attribute list side bar from #527 pops up. 
**Then** She can use a special note field to add bullet points. 
**and** each bullet point remembers what metrics were active while making that observation

**When** a user has created at least one bullet point for a building or package
**Then** that building gets a special marker in the 3D Codemap, so the user can see that he added a note for that building. 

**Given** a user has created at least one bullet point for a building
**When** she clicks on that building
**Then** the attribute list side bar shows each bullet and (on hover) what metrics were active for each bullet. 

**Given** a user has created at least one bullet point for a building
**When** she exports the cc.json
**Then** she can select if she wants to include these notes in the export. The default is no, to prevent rough comments to leak :)

What would the cc.json look like after adding this notes field? Is it stored for each node or is it stored globally or ...?

```json
{
  "nodes":[],
  "edges":[],
  "comments": []
```

**Comment by RomanenkoVladimir** [comment]
@BridgeAR I think this would be a great UX improvment! 
We should implement that soonish, what do you say?

**Comment by BridgeAR** [comment]
I agree. But let's first fix multiple bugs and finish a couple of features that we started :)

**Comment by ce-bo** [comment]
Are still any questions open?

**Comment by ChristianHuehn** [comment]
Does not fit the current vision for codecharta nor does it represent how we work with it currently. 

This will be removed for now
